marda_extractors_api/__init__.py

```diff
@@ -395,25 +402,43 @@ def apply_template_args(
         input_path: Path,
         output_type: str | None = None,
         output_path: Path | None = None,
-    ):
+        additional_template: dict | None = None,
+    ) -> str:
         """Reference implementation of templating in MaRDA Metadata Extractor Schemas.
 
         See the :doc:`mme_schema:mme_schema/UsageTemplate` for details of the individual arguments.
+
+        Parameters:
+            command: The command to apply the template to.
+            method: The execution method to use.
+            input_type: The input type to use.
+            input_path: The input path to use.
+            output_type: The output type to use.
+            output_path: The output path to use.
+            additional_template: Additional template arguments to use, which
+                overwrite the default arguments.
+
+        Returns:
+            The templated command.
+
         """
-        if method == SupportedExecutionMethod.CLI:
-            command = command.replace("{{ input_type }}", f"marda:{input_type}")
```

Comment:
I'm not sure why we were prefixing with `marda` here @PeterKraus, but this might break yadg now. Seems sensible that yadg should announce that it requires `marda:biologic-mpr` to be passed in its own template.

Reply:
It ought to work without the `marda:` prefix, I add it if the plain `ID` is not found among supported filetypes.